validator/accounts/accounts_backup_test.go

```diff
@@ -20,8 +20,8 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/testutil"
 	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
 	"github.com/prysmaticlabs/prysm/shared/testutil/require"
-	"github.com/prysmaticlabs/prysm/validator/accounts/v2/wallet"
-	v2keymanager "github.com/prysmaticlabs/prysm/validator/keymanager/v2"
+	"github.com/prysmaticlabs/prysm/validator/accounts/wallet"
+	keymanager2 "github.com/prysmaticlabs/prysm/validator/keymanager"
```

Comment:
We can probably get rid of this ugly alias everywhere by renaming local variables.